exercises/intro/README.md

```diff
@@ -145,6 +149,27 @@ There are 4 distinct functions that are entangled in this statement-based
 program. No single function can be tested in isolation. Let's pull them apart:
 
 ```scala
+def iterate(max: Int): List[Int] = ???
+def filterEven(xs: List[Int]): List[Int] = ???
+def square(xs: List[Int]): List[Int] = ???
+def sum(xs: List[Int]): Int = ???
```

Comment:
Good trick 👍  I didn't think about using `???` to show the intermediary step here.